interactive-cherry-pick/interactive-cherry-pick.sh

```diff
@@ -60,7 +60,9 @@ git cherry -v \
 
 # Remove the leading plus signs and trim the sha1s in the file to 8 characters,
 # to match what "git rebase -i" outputs.
-sed -i.backup -e 's/^+ \(\S\S\S\S\S\S\S\S\)\S*/\1/' "$cp_list"
+# NOTE: Take care to ensure that changes to this still work on macOS, which uses
+# BSD sed instead of GNU sed.
+sed -i.backup -re 's/^\+ ([a-fA-F0-9]{8})[a-fA-F0-9]*/\1/' "$cp_list"
```

Comment:
I made a dummy commit in my branch and tried this out. This is what I saw:

`+ 71b8324bfe0c098e3c2cb7d3bd4ba8028bbae5b3 feat(player): Dummy commit`

There was also an error in the terminal, saying `fatal: bad revision '+'`.
So it looks like this isn't working as intended.

Reply:
Do you mean you tried that with the PR?  Or without?

Try this:

```sh
echo "+ 71b8324bfe0c098e3c2cb7d3bd4ba8028bbae5b3 feat(player): Dummy commit" \
  | sed -re 's/^\+ ([a-fA-F0-9]{8})[a-fA-F0-9]*/\1/'
```

Does it format the input correctly on your mac?

Reply:
I double-checked, and this works on the mac in our lab.

Reply:
Oh... I checked again and I had forgotten to apply this PR before testing. Whoops!